W-14288798-Associate-BGs-environments (#158)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ps-config-env.adoc

By default, a private space is associated with all sandbox environments in its business groups.

== Associate Business Groups and Environments

If you have the CloudHub Network Administrator permission, associate environments in the business group or groups with the private space:

//SELECT PRIVATE SPACE SHARED
include::partial$select-private-space.adoc[tag=selectPrivateSpace]
include::partial$select-private-space.adoc[tag=clickPrivateSpaceName]
. Click the *Environments* tab to:
** <<associate-business-groups>>
** <<associate-environments>>

=== Associate Business Groups

. Select the business groups from the drop-down list:  
+
--
** *This business group and its children*
+ 
This option allows you to select either all environments, all sandbox environment or all production environments for the selected business group and all its children.
** *Selected business groups*
+
Enter the business group name in the *Add business group* field and select it from the list. This option allows you to select specific environments for specific child business groups.
+

> The list includes the business group and its child business groups to which the private space is registered and that have not yet been added.

Reviewer: [start=2]
. Click *Save Changes* or *Discard Changes* to discard or confirm the association.

> The list includes the business group and its child business groups to which the private space is registered and that have not yet been added.

Reviewer: ```suggestion
The list includes the business group and its child business groups to which the private space is registered and that have not yet been added.

[start=2]
. Click *Save Changes* or *Discard Changes* to discard or confirm the association.
```

--
+
image::ps-select-business-group.png[Select the business group from the drop-down list.]

=== Associate Environments

. Select the environments from the drop-down list:
+
--
** *All Environment*
+
Associates the private space with all sandbox and production environments in the business group and all its children, or in specific child business organizations.
** *All Sandbox Environments*
+
Associates the private space with only sandbox environments in the business group and all its children, or in specific child business organizations.
** *All Production Environments*
+
Associates the private space with only production environments in the business group and all its children, or in specific child business organizations.
** *Select Environments Manually*
+
Allows you to associate the private space with specific environments in specific child business groups.
+
*** Check the *Sandbox* or *Production* box to bulk select all sandbox or production environments in specific child business organizations.

> *** Check the box for each specific subenvironment to individually select the desired environments in specific child business organizations.

Reviewer: ```suggestion
*** Check the box for each specific subenvironment to individually select the desired environments in specific child business organizations.

[start=2]
. Click *Save Changes* or *Discard Changes* to discard or confirm the association.
```

--
+
image::ps-select-environment.png[Select the environments from the drop-down list.]

> === Finish Your Association

Reviewer: ```suggestion
=== Add or Remove Business Groups
```

> You can add new busines groups using the *Add business group* field.

Reviewer: ```suggestion
You can add or remove business groups for your private space:

To add new business groups, use the *Add business group* field.

To remove a business group, click the trash can icon for the entry.
```

> To remove a business group, click the trash can icon for the entry.

Reviewer: ```suggestion

```

image::bg-delete.png[Delete icon on the Environments tab]

> Click *Save Changes* or *Discard Changes* to discard or confirm the association.

Reviewer: ```suggestion

```

For information, see xref:ch2-deploy-private-space.adoc[].

You can associate a private space with the production and sandbox environments in the business group, and its child business groups, that it's registered with or with a subset of business groups.

By default, a private space is associated with all sandbox environments in its business groups.

== Associate Business Groups and Environments

If you have the CloudHub Network Administrator permission, associate environments in the business group or groups with the private space:

//SELECT PRIVATE SPACE SHARED
include::partial$select-private-space.adoc[tag=selectPrivateSpace]
include::partial$select-private-space.adoc[tag=clickPrivateSpaceName]
. Click the *Environments* tab to:
** <<associate-business-groups>>
** <<associate-environments>>

=== Associate Business Groups

. Select the business groups from the drop-down list:  
+
--
** *This business group and its children*
+ 
This option allows you to select either all environments, all sandbox environment or all production environments for the selected business group and all its children.
** *Selected business groups*
+

> Enter the business group name in the *Add business group* field and select it from the list. This option allows you to select specific environments for specific child business groups.

Reviewer 2: `select specific environments for specific child business groups` -> You can select specific environments for root org as well as it's children. So, can we change this to `specific business groups`

+
The list includes the business group and its child business groups to which the private space is registered and that have not yet been added.

[start=2]
. Click *Save Changes* or *Discard Changes* to discard or confirm the association.
--
+
image::ps-select-business-group.png[Select the business group from the drop-down list.]

=== Associate Environments

. Select the environments from the drop-down list:
+
--
** *All Environment*
+

> Associates the private space with all sandbox and production environments in the business group and all its children, or in specific child business organizations.

Reviewer 2: `specific child business organizations` -> same as above. You can select all environments for root org as well as it's children. So, can we change this to `specific business groups`. This should apply to all environment types.
